src/merge_sort.py

```diff
@@ -0,0 +1,61 @@
+""" This module will implement a merge sort on a list"""
+# This solution was heavily influenced by the example found on interactive
+# python found at the URL below. I've attempted to make the function a class
+# with a recursive method:
+# http://interactivepython.org/runestone/static/pythonds/SortSearch/TheMergeSort.html
+import timeit
+import random
+
+
+class MergeSort(object):
+
+    def merge_sort(self, merge_list):
+        """This function will call the recursive merge sort method and returb a
+        sorted list"""
+        if len(merge_list) > 1:
+            mid_point = len(merge_list) // 2
+            left = merge_list[:mid_point]
+            right = merge_list[mid_point:]
+
+        self._merge_sort_recursive(merge_list, left, right)
```

Comment:
You're vulnerable to a `NameError` here; make sure you're testing on lists of 1 or 0 items.